package.json

```diff
@@ -20,17 +20,13 @@
   "dependencies": {
     "boom": "^2.6.1"
   },
-  "devDependencies": {
-    "coffee-script": ">=1.7.x",
-    "mocha": "~1.x.x",
-    "chai": "^1.10.0",
-    "express": "^4.10.5",
-    "namespaced-request": "^1.3.1"
-  },
   "scripts": {
-    "compile": "coffee --bare --compile --output lib/ src/",
-    "prepublish": "npm run compile",
-    "pretest": "npm run compile",
     "test": "mocha"
+  },
+  "devDependencies": {
```

Comment:
I imagine you know this, but you're missing `@goodeggs/toolkit`, `typescript`, babel, etc.